packages/system/hcloud/charts/hcloud-cloud-controller-manager/templates/daemonset.yaml

```
@@ -0,0 +1,108 @@
+  namespace: {{ .Release.Namespace }}
+spec:
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      {{- include "hcloud-cloud-controller-manager.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "hcloud-cloud-controller-manager.selectorLabels" . | nindent 8 }}
+        {{- if .Values.podLabels }}
+        {{- toYaml .Values.podLabels | nindent 8 }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)